CMakeLists.txt

```diff
@@ -87,8 +87,9 @@ ENDIF(APPLE)
 ENDIF(MINGW)
 
 #-----------------------------------------------------------------------------
-# Search for Qt5. We need the latest version of Qt
-SET(QT_MIN_VERSION "5.2.1")
+# Search for Qt5. We need the version available in the last two Ubuntu LTS
+# Releases
+SET(QT_MIN_VERSION "5.5.1")
```

Comment:
Leave the minimum version at 5.2.1. We want to be backward compatible as much as possible. I don't see a good reason why 5.5.1 should be minimum rather than that it's shipped with Ubuntu 16.04.

Reply:
I reverted the entire commit, so CMakeLists.txt stays untouched.